src/cryptonote_config.h

```diff
@@ -190,8 +190,8 @@ static constexpr hardfork_conf FORK_CONFIG[] = {
 struct common_config
 {
 	static constexpr uint64_t POISSON_CHECK_TRIGGER = 10;  // Reorg size that triggers poisson timestamp check
-	static constexpr uint64_t POISSON_CHECK_DEPTH = 150;   // Main-chain depth of the poisson check. The attacker will have to tamper 50% of those blocks
-	static constexpr double POISSON_LOG_P_REJECT = -100.0; // Reject reorg if the probablity that the timestamps are genuine is below e^x, -100 = 10^-44
+	static constexpr uint64_t POISSON_CHECK_DEPTH = 60;   // Main-chain depth of the poisson check. The attacker will have to tamper 50% of those blocks
```

Comment:
Is this `60` related somehow to https://github.com/ryo-currency/ryo-currency/pull/15/files#diff-75dbb84a608233b878963350f63597e1R213 or is it fortune that it is DIFFICULTY_WINDOW_V3 - 1

Reply:
Mostly coincidence, I had to reduce the check depth a lot and this is the number that came to my head.